lib/instrumentation/modules/tedious.js

```diff
@@ -69,7 +69,7 @@ module.exports = function (tedious, agent, { version, enabled }) {
         if (!request.parametersByName) {
           return super.makeRequest(...arguments)
         }
-        const span = agent.startSpan(null, 'db', 'mssql', 'query')
+        const span = ins.createSpan(null, 'db', 'mssql', 'query')
```

Comment:
Just noting out loud that, unlike some of the other instrumentations, it looks like we're now creating this span but never binding it to a particular run context.  I'm presuming this is intentional.

Reply:
Yes. 
- The tedious API's user callback was already wrapped in a `ins.bindFunction` which by default picks up the current run context, and
- currently the wrapped callback doesn't handle captureError or setting span.outcome, so there is no *current* need to have that run in the *span's* run-context as has been done for other instrumentations. I've open https://github.com/elastic/apm-agent-nodejs/issues/2493 to improve the MSSQL instrumentation here.

One minor thing is that the `super.makeRequest(...arguments)` is *not* currently wrapped in `ins.withRunContext` to make it run in the span's run context. That *could* be done, but given that for #2103 this will be marked an exit span and nothing runs in the span context, there can't be any children. In other words, I'm not aware of any harm in not wrapping that `super.makeRequest` in the span's run context, rather than in its parent (the current run context).